bin/build_frontend.sh

```diff
@@ -5,29 +5,18 @@ set -eo pipefail
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-# Builds the React frontend, *only* if we are in CI.
+# Builds the React frontend.
 
-# True if the environment variable 'CI' is NOT empty.
-# In CircleCI, this variable is "true" and in local dev, it's "".
-if [[ ! -z "${CI}" ]]; then
```

Comment:
This whole CI/not-CI thing shaves 18s off of building the image for me. However, it creates this situation where the local development environment is wildly different than what we build in CI for stage/prod which makes it really hard to figure out what's going on when debugging static files and what's actually getting built.

I wasted a whole day today. I think it's worth losing 18s per build to have a local dev environment building images that are what we're building for stage/prod.